### DISCUSSION TITLE
Why is MaxJumpTime multiplied by 4/3 ?

### DISCUSSION THREAD

**Discussion by themorfeus** [ProjectBorealis/PBCharacterMovement#47] [Q&A]
In `APBPlayerCharacter::BeginPlay` a variable called MaxJumpTIme is set - it's commented as being set to the 'Max Jump time to get to the top of the arc', however the value is ultimately multplied by four thirds, and i can't figure out why. Is there a reason for this magic number, or did that 'just work' ?

**Comment by mastercoms** [comment]
Hi, though I can't remember the exact derivation, this is a linear estimate for gravity acceleration. It only works for smaller periods of time and then begins to underestimate, but this is okay because the value is used to avoid jump spam up slopes which only happens with low jump heights.
